homeassistant/components/fitbit/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration has no actions.
+  appropriate-polling:
+    status: done
+    comment: |
+      The integration uses polling to get data from Fitbit API every 30 minutes.
+  brands: done
+  common-modules:
+    status: done
+    comment: Coordinator is in coordinator.py and there is no base entity.
+  config-flow:
+    status: done
+    comment: The integration uses an OAuth flow setup from the UI.
+  config-flow-test-coverage: done
+  dependency-transparency: done
```

Comment:
The dependency is not built and published in a CI

Reply:
Thanks, forgot that I don't own this.